.github/workflows/on-pr.yml

```diff
@@ -18,6 +23,49 @@ jobs:
       - name: 'Dependency Review'
         uses: actions/dependency-review-action@v4
 
+  test_web:
```

Comment:
This workflow channel already contains a `test_e2e_web` step that uses the get-affected step as an input. We should consolidate the steps and make them all mandatory.

Reply:
@SokratisVidros Can you clarify what your looking for when you say make them all mandatory?

They are already all mandatory by github branch rules?